### PR TITLE
assignment3@mlscmoulika

### PR DESCRIPTION
**Pull request by mlscmoulika**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Comment by aravindvnair99** [comment]
@mlscmoulika Next time fill up the pull request template properly.

File: Moulika/ass3.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form action="">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Form is missing identifier.

File: Moulika/ass3.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="First_name">First Name</label>
+        <input type="text" placeholder="First Name">
+        <label for="Last_name">Last Name</label>
+        <input type="text" placeholder="Last Name">
+        <br/><br/>
+        <label for="Gender_Male">Gender: Male:</label>
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: **Gender** won't be inside label. Only the types will be.

File: Moulika/ass3.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <input type="radio">
+        <label for="Gender_Female">Female</label>
+        <input type="radio">
+        <label for="Gender_Others">Others</label>
+        <input type="radio">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: For should be linked with `id` which is missing for the inputs.

File: Moulika/ass3.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="" id="" placeholder="Email">
+        <label for="Password">Password:</label>
+        <input type="password" name="" id="">
+        <label for="birthday">Birthday</label>
+        <input type="date" name="" id="">
+        <br/>
+        <label for="terms">I agree to terms and conditions </label>
+        <input type="checkbox" name="" id="">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: `required` attribute is missing

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Moulika/ass3.html (hunk 1 above)
Comment (aravindvnair99) [reply Moulika/ass3.html]: @mlscmoulika This is still pending.

File: Moulika/ass3.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        <label for="Gender">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Gender">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password">
+        <label for="birthday">Birthday</label>
+        <input type="date" name="date" id="date">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Missing `autocomplete`. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by mlscmoulika** [comment]
I am extremely sorry for the delay .
Will not repeat it .
Resubmitted today please check.

File: Moulika/ass3.html (hunk 1 above)
Comment (aravindvnair99) [reply Moulika/ass3.html]: @mlscmoulika Still pending

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by VinayakTekade — DISMISSED** [review]

File: Moulika/ass3.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form>
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Form is missing identifier.

File: Moulika/ass3.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        <label for="First_name">First Name</label>
+        <input type="text" placeholder="First Name" required autocomplete="on">
+        <label for="Last_name">Last Name</label>
+        <input type="text" placeholder="Last Name"required autocomplete="on">
+        <br/><br/>
+        <label for="Gender">Gender</label>
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Gender won't be inside the label. Only the types will be.

File: Moulika/ass3.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+        <label for="Gender">Male</label>
+        <input type="radio"name ="Gender"id="Male"autocomplete="on">
+        <label for="Gender">Female</label>
+        <input type="radio"name="Gender"id ="Female"autocomplete="on">
+        <label for="Gender">Others</label>
+        <input type="radio"name ="Gender"id="Others"autocomplete="on">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Incorrect syntax. `for` corresponds to `id`. Check `label` syntax.

File: Moulika/ass3.html [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email"autocomplete="on">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password"autocomplete="on">
+        <label for="birthday">Birthday</label>
+        <input type="date" name="date" id="date"autocomplete="on">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Use specific `autocomplete`. Not general. Set the `autocomplete` type.

File: Moulika/ass3.html [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+        <br/>
+        <label for="terms">I agree to terms and conditions </label>
+        <input type="checkbox" name="terms_agree" id="terms_agree"autocomplete="on">
+        <br/>
+        <label for="submit"></label>
+        <input type="submit" value="Submit" autocomplete="on">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: `autocomplete` won't be applicable here.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@mlscmoulika Please go through syntaxes and applications of the tags before implementing them.

- [HTML \<label> Tag - w3schools](https://www.w3schools.com/tags/tag_label.asp)
- [The HTML autocomplete attribute - MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

File: Moulika/ass3.html [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="Register">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="Register"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email"autocomplete="email">
+        <label for="Register">Password:</label>
+        <input type="password" name="password" id="password"autocomplete="new-password">
+        <label for="Register">Birthday</label>
+        <input type="date" name="date" id="date"autocomplete="Birthday">
+        <br/>
+        <label for="Register">I agree to terms and conditions </label>
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Incorrect usage of `label`. `for` attribute corresponds to the `id`.

File: Moulika/ass3.html [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="Register">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Register">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="Register"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email"autocomplete="email">
+        <label for="Register">Password:</label>
+        <input type="password" name="password" id="password"autocomplete="new-password">
+        <label for="Register">Birthday</label>
+        <input type="date" name="date" id="date"autocomplete="Birthday">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Wrong `autocomplete`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Moulika/ass3.html [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email"autocomplete="email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password"autocomplete="new-password">
+        <label for="Birthday">Birthday</label>
+        <input type="date" name="date" id="date">
+        <br/>
+        <label for="terms_agree">I agree to terms and conditions </label>
+        <input type="checkbox" name="terms_agree" id="terms_agree" required autocomplete="checkbox">
```
Comment (aravindvnair99) [inline Moulika/ass3.html:32]: There is no `autocomplete` for a checkbox. Remove it.

File: Moulika/ass3.html [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="Female">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Others">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email"autocomplete="email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password"autocomplete="new-password">
+        <label for="Birthday">Birthday</label>
+        <input type="date" name="date" id="date">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Missing `autocomplete`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Moulika/ass3.html [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form id="Register">
+        <label for="First name">First Name</label>
+        <input type="text" placeholder="First Name" required autocomplete="given-name">
+        <label for="Last name">Last Name</label>
+        <input type="text" placeholder="Last Name"required autocomplete="family-name">
```
Comment (aravindvnair99) [inline Moulika/ass3.html:15]: Incorrect usage of `label`. `for` attribute corresponds to the `id`. Refer this: [HTML \<label> Tag](https://www.w3schools.com/tags/tag_label.asp)

File: Moulika/ass3.html [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="Others">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password">
+        <label for="Birthday">Birthday</label>
+        <input type="date" name="date" id="date" autocomplete="bday">
+        <br/>
+        <label for="terms_agree">I agree to terms and conditions </label>
```
Comment (aravindvnair99) [inline Moulika/ass3.html:31]: Missing `required`.

File: Moulika/ass3.html [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+        <label for="Male">Male</label>
+        <input type="radio"name ="Gender"id="Male">
+        <label for="Female">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Others">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password">
```
Comment (aravindvnair99) [inline Moulika/ass3.html:27]: Missing `autocomplete`.

File: Moulika/ass3.html [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+        <br/><br/>
+        Gender: 
+        <label for="Male">Male</label>
+        <input type="radio"name ="Gender"id="Male">
+        <label for="Female">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Others">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email">
```
Comment (aravindvnair99) [inline Moulika/ass3.html]: Missing `autocomplete`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Moulika/ass3.html (hunk 15 above)
Comment (VinayakTekade) [reply Moulika/ass3.html:15]: Still incomplete.

File: Moulika/ass3.html (hunk 17 above)
Comment (VinayakTekade) [reply Moulika/ass3.html:27]: Still incomplete.

File: Moulika/ass3.html (hunk 16 above)
Comment (VinayakTekade) [reply Moulika/ass3.html:31]: Still incomplete.

File: Moulika/ass3.html (hunk 13 above)
Comment (VinayakTekade) [reply Moulika/ass3.html:32]: Still incomplete.

**Review by VinayakTekade — APPROVED** [review]

**Comment by VinayakTekade** [comment]
Complete all the changes required in another pull request.

**Review by aravindvnair99 — COMMENTED** [review]
@mlscmoulika Merging and closing the pull request for now. Make the necessary changes in another pull request and send it over from a different branch. Also, make sure to fill up the pull request template and also refactor the code henceforth.

**Pull request merged**